### ISSUE TITLE
Copier

### ISSUE THREAD

**Issue by PocketKiller** [CreeperLewisUK/DrawIt#1]
why did you copy my plugin? delete this repo please Thanks.

**Comment by PocketKiller** [comment]
this is completely made by me, you did you do anything.

**Comment by PocketKiller** [comment]
@majovmc don't star this copy.

**Comment by majovec** [comment]
@PocketKiller what copy

**Comment by PocketKiller** [comment]
this whole plugin is copied, I made it, i made a pocketmine thread with the code, then he took it and create a repo. so this is WHOLE A COPY.

**Comment by majovec** [comment]
seriously :o

**Comment by majovec** [comment]
@PocketKiller  and when thou shalt make continuation of Simon Says

**Comment by PocketKiller** [comment]
yes I am serious, I did not upload plugin.yml so he did not be able to make it, copier.
oh I'll do SimonSays after I fully finish DrawMyThing, I am at %91

**Comment by majovec** [comment]
but you do not have it on github

**Comment by PocketKiller** [comment]
its not on github, It's private. do you want me to upload it?

**Comment by PocketKiller** [comment]
actually I am submitting it on pocketmine

**Comment by majovec** [comment]
But you do not have to publicly be sent in the mail ?

**Comment by PocketKiller** [comment]
I've done the plugin in private, which doesn't have customizablity of postions yet.

**Comment by majovec** [comment]
ok

**Comment by majovec** [comment]
and this your email ? pocketkiller@imagicalmine.me

**Comment by PocketKiller** [comment]
yes.

**Comment by majovec** [comment]
i send you message you plz read email

**Comment by PocketKiller** [comment]
Okay.

**Comment by majovec** [comment]
thanks

**Comment by PocketKiller** [comment]
this place has turned into a chat room :0

**Comment by majovec** [comment]
Exactly At least he 's got spam :D

**Comment by PocketKiller** [comment]
hello?

**Comment by majovec** [comment]
Hello :D
